package.json

```diff
@@ -23,9 +23,9 @@
     "standard": "^14.3.1"
   },
   "dependencies": {
-    "ipfs-unixfs-importer": "^0.44.1",
-    "ipld": "^0.25.3",
-    "ipld-in-memory": "^3.0.2"
+    "ipfs-unixfs-importer": "^2",
+    "ipld": "^0",
```

Comment:
What does `^0` mean?

Reply:
avoid install multi versions in node_modules

Reply:
I mean what versions of ipld would this allow to be installed?

Reply:
lastset